applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -112,6 +112,10 @@ config BT_VCS
 	bool
 	default y
 
+config BT_PACS_SRC_CONTEXT
```

Comment:
Just for my understanding: Headsets use PACS source and not sink?

Reply:
My bad, I should enable PACS but set PACS_SRC_CONTEXT to 0x0000